BetterShardsBukkit/src/main/java/vg/civcraft/mc/bettershards/misc/CustomWorldNBTStorage.java

```diff
@@ -54,7 +69,12 @@ public void save(EntityHuman entityhuman) {
 			NBTCompressedStreamTools.a(nbttagcompound, output);
 
 			// Now to run our custom mysql code
-			db.savePlayerData(uuid, output, getInvIdentifier(uuid), sect.get(uuid));
+			if (!BetterShardsPlugin.getTransitManager().isPlayerInTransit(uuid)) {
+				db.savePlayerData(uuid, output, getInvIdentifier(uuid), sect.get(uuid));
+			}
+			else {
+				logger.log(Level.WARNING, "Didn't save data for {0}, because he was in transit", entityhuman.getName());
```

Comment:
So a gotcha here; `'` is parsed as "read everything that follows as a string literal and don't interpret". That's why if you look at my other log lines I never use apostraphes or contractions. So easy fix; just "Did not save..." and that's it.